src/main/java/net/snowflake/client/jdbc/SnowflakeParameterMetadata.java

```diff
@@ -15,10 +15,11 @@
  * describe sql response.
  */
 class SnowflakeParameterMetadata implements ParameterMetaData {
-  private SFStatementMetaData statementMetaData;
+  private SFPreparedStatementMetaData statementMetaData;
```

Comment:
can you refactor this variable name also?

Reply:
Refactored